torch/linalg/__init__.py

```diff
@@ -602,34 +602,32 @@
 cond = _add_docstr(_linalg.linalg_cond, r"""
 linalg.cond(input, p=None, *, out=None) -> Tensor
 
-Computes the condition number of a matrix :attr:`input`,
-or of each matrix in a batched :attr:`input`, using the matrix norm defined by :attr:`p`.
+Computes the condition number of a matrix :attr:`input`, or of each matrix in a batched :attr:`input`,
+using the matrix norm defined by :attr:`p`.
+
 For norms ``p = {'fro', 'nuc', inf, -inf, 1, -1}`` this is defined as the matrix norm of :attr:`input`
-times the matrix norm of the inverse of :attr:`input`. And for norms ``p = {None, 2, -2}`` this is defined as
-the ratio between the largest and smallest singular values.
+times the matrix norm of the inverse of :attr:`input` computed using :func:`torch.linalg.norm`. While
+for norms ``p = {None, 2, -2}`` this is defined as the ratio between the largest and smallest singular
+values computed using :func:`torch.linalg.svd`.
+
+This function supports ``float``, ``double``, ``cfloat`` and ``cdouble`` dtypes.
 
-This function supports ``float``, ``double``, ``cfloat`` and ``cdouble`` dtypes for :attr:`input`.
-If the input is complex and neither :attr:`dtype` nor :attr:`out` is specified, the result's data type will
-be the corresponding floating point type (e.g. float if :attr:`input` is complexfloat).
+.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU.
 
-.. note:: For ``p = {None, 2, -2}`` the condition number is computed as the ratio between the largest
-          and smallest singular values computed using :func:`torch.linalg.svd`.
-          For these norms :attr:`input` may be a non-square matrix or batch of non-square matrices.
+.. note:: For ``p = {None, 2, -2}`` :attr:`input` may be a non-square matrix or batch of non-square matrices.
           For other norms, however, :attr:`input` must be a square matrix or a batch of square matrices,
           and if this requirement is not satisfied a RuntimeError will be thrown.
 
 .. note:: For ``p = {'fro', 'nuc', inf, -inf, 1, -1}`` if :attr:`input` is a non-invertible matrix then
           a tensor containing infinity will be returned. If :attr:`input` is a batch of matrices and one
           or more of them is not invertible then a RuntimeError will be thrown.
 
-.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU.
-
 Args:
     input (Tensor): the input matrix of size :math:`(m, n)` or the batch of matrices of size :math:`(*, m, n)`
-                    where `*` is one or more batch dimensions.
+                    where :math:`*` is one or more batch dimensions.
```

Comment:
I suggest using just \` \`, without :math:. Inline math looks odd to me, it is more useful for standalone equations.
With just \` \` it would look like this
![image](https://user-images.githubusercontent.com/19621411/106795540-df0e0c00-6662-11eb-8983-ac118a187e92.png)
This example is taken from current `torch.svd` docs.